libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/roomlist/RoomList.kt

```diff
@@ -23,25 +23,34 @@ import kotlinx.coroutines.withTimeout
 import timber.log.Timber
 import kotlin.time.Duration
 
-interface RoomSummaryDataSource {
-
+/**
+ * Holds some flows related to a specific set of rooms.
+ * Can be retrieved from [RoomListService] methods.
+ */
+interface RoomList {
```

Comment:
`RoomListState`? Otherwise it looks like something we can interact with, instead of just holding state.

Reply:
Mmmm, we already have a State inside, so might be a bit weird...